src/attacks/attackconsole.py

```diff
@@ -168,6 +168,8 @@ def do_run(self, arg):
             print("*** Exception: {e}".format(e=e))
             log_dict.update(event="attack_failed", failed_with=str(e).replace("\n", " "))
             self.logger.info("Attack failed", log_dict=log_dict)
+        except KeyboardInterrupt:
+            self.attack.handle_interrupt()
```

Comment:
Did you manually test that the attackconsole successfully returns to the input prompt for all cases in handle_interrupt()?

Reply:
What I mean is: There are three cases handle_interrupt(), did you test at least one attack for each of these cases successfully? Else we have untested code in there.

Reply:
Sorry, answered all the comments while I was in a bit of a hurry yesterday ^^ yes I tested every possible code branch, just not with all possible attacks